src/System.Net.WebClient/tests/WebClientTest.cs

```diff
@@ -18,7 +18,7 @@ namespace System.Net.Tests
     public class WebClientTest
     {
         [Fact]
-        public static void DefaultCtor_PropertiesReturnExpectedValues()
+        public static void WebClient_DefaultCtor_PropertiesReturnExpectedValues()
```

Comment:
Do you have an occurrence of where a report is not including the full method name? This is done by XUnit and it always logs everything as Namespace.Class.Method, so you shouldn't include the class name in the test name as it will be duplicated in the logs and that is not the guidelines we follow.